install-scripts/windows/wsl-install.sh

```diff
@@ -0,0 +1,50 @@
+#Updated Script file
```

Comment:
There shouldn't be any comments about `#!/bin/bash`. The kernel expects it to be the first line to be able to register the file as a script.

Reply:
@SatyamRaina - Still need to fix this issue.